addon/mixins/table-header.js

```diff
@@ -105,6 +107,19 @@ export default Mixin.create({
     return this.getProperties(['iconAscending', 'iconDescending']);
   }).readOnly(),
 
+  init() {
+    this._super(...arguments);
+
+    let fixed = this.get('fixed');
+    let height = this.get('sharedOptions.height');
+
+    warn(
```

Comment:
Shouldn't be necessary as `warn` is supposed to be an empty function in a production build. I have not actually verified this though.

https://www.emberjs.com/api/#method_warn